<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab2SockeClient</class>
 <widget class="QWidget" name="Tab2SockeClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="8,1,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>수신 데이터</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBRecvDataClear">
       <property name="text">
        <string>수신 삭제</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBServerConnect">
       <property name="text">
        <string>서버 연결</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextEdit" name="pTERecvData"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,6,1">
     <item>
      <widget class="QLineEdit" name="pLERecvId"/>
     </item>
     <item>
      <widget class="QLineEdit" name="pLESendData"/>
     </item>
     <item>
      <widget class="QPushButton" name="pPBSendButton">
       <property name="text">
        <string>송신</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pPBRecvDataClear</sender>
   <signal>clicked()</signal>
   <receiver>pTERecvData</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
